intranet/settings/__init__.py

```diff
@@ -42,7 +42,8 @@
 ENABLE_WAITLIST = False  # WARNING: Enabling the waitlist causes severe performance issues
 ENABLE_BUS_APP = True
 ENABLE_BUS_DRIVER = True
-ENABLE_PRE_EIGHTH_REDIRECT = False
+ENABLE_PRE_EIGHTH_SIGNUP_REDIRECT = False
```

Comment:
If we're in the business of renaming this option, I would propose instead `ENABLE_PRE_EIGHTH_CLOSE_SIGNUP_REDIRECT`.

Reply:
Done